th-TH.json

```diff
@@ -1,86 +1,10 @@
 {
   "1 {0} = {1} {2}": "1 {0} = {1} {2}",
-  "APR": "APR",
-  "Action": "Action",
-  "Batched At": "Batched At",
```

Comment:
Your PR is deleted many words :( Please help me revert them  before merging

Reply:
I just noticed that there is a new template of the JSON file. I will fix it and resubmit again.